src/main/java/org/junit/internal/Throwables.java

```diff
@@ -73,7 +73,7 @@ public static String getStacktrace(Throwable exception) {
     public static String getTrimmedStackTrace(Throwable exception) {
         List<String> trimmedStackTraceLines = getTrimmedStackTraceLines(exception);
         if (trimmedStackTraceLines.isEmpty()) {
-            return getFullStackTrace(exception);
+            return Throwables.getStacktrace(exception);
```

Comment:
Thanks for deleting the redundant `getFullStackTrace()` method. No need to specify `Throwables` here since it's in the same class.